src/sql/platform/query/common/queryManagement.ts

```diff
@@ -235,6 +241,11 @@ export class QueryManagementService implements IQueryManagementService {
 			return runner.disposeQuery(ownerUri);
 		});
 	}
+	public setQueryExecutionOptions(ownerUri: string, options: azdata.QueryExecutionOptions): Thenable<void> {
+		return this._runAction(ownerUri, (runner) => {
+			return runner.setQueryExecutionOptions(ownerUri, options);
```

Comment:
we need to null check `runner.setQueryExecutionOptions` since some providers, such as Postgres, may not implement.